CONTRIBUTING.md

```diff
@@ -82,16 +82,26 @@ The test module, clusterctl, and experiments do not provide any backward compati
 
 #### Backporting a patch
 
-We only accept backports of critical bugs, security issues, or bugs without easy workarounds, any
-backport MUST not be breaking for either API or behavioral changes. In order to improve user/developer experience
-maintainers can choose to backport:
-- Doc improvements
+We generally do not accept PRs directly against release branches, while we might accept backports of fixes/changes already
+merged into the main branch.
+
+Any backport MUST not be breaking for either API or behavioral changes. 
+
+We generally allow backport of following changes to all the supported branches:
+- Critical bugs fix, security issues fix, or fix for bugs without easy workarounds.
+- Dependency bumps for CVE (usually limited to CVE resolution; backports of non-CVE related version bumps are considered exceptions to be evaluated case by case)
+- Cert-manager version bump (to avoid having branch using cert-manager versions out of support, when possible)
```

Comment:
```suggestion
- Cert-manager version bumps (to avoid having releases with cert-manager versions that are out of support, when possible)
```
nit: sorry I know it was like this before. Just while we're already here